pkg/fleet/installer/service/datadog_installer.go

```diff
@@ -181,7 +181,16 @@ func getAgentIDs() (uid, gid int, err error) {
 
 // startInstallerStable starts the stable systemd units for the installer
 func startInstallerStable(ctx context.Context) (err error) {
-	return startUnit(ctx, installerUnit)
+	_, err = os.Stat("/etc/datadog-agent/datadog.yaml")
```

Comment:
This doesnt' work on [MacOS](https://github.com/DataDog/datadog-agent/blob/03aedd2c7b82c9bd7b42512b8542e1e3a9cc347b/cmd/agent/common/path/path_darwin.go#L16) and on [FreeBSD](https://github.com/DataDog/datadog-agent/blob/03aedd2c7b82c9bd7b42512b8542e1e3a9cc347b/cmd/agent/common/path/path_freebsd.go#L16) but the build flag exclude only Windows (`//go:build !windows`).

Reply:
Those platforms aren't supported by the installer today so it's fine 👍🏻